dev/puppet-server.conf.sample

```diff
@@ -115,3 +115,8 @@ authorization: {
         }
     ]
 }
+
+# general puppetserver settings
+puppetserver: {
+    environment-class-cache-enabled: true
+}
```

Comment:
@rlinehan Ah, ok.  That makes sense.  Adding the commented out settings for `code-*-command` to a `versioned-code` section of the dev config seems reasonable to me.